### PR TITLE
adaption of IdTypes

### PR DESCRIPTION
**Pull request by ue71603**
Each Id has now its own type

### CODE REVIEW

**Comment by ue71603** [comment]
> I think that at least the following id's would also have to be updated: _FromLegIdRef_, _ToLegIdRef_ (OJP_FareSupport.xsd and OJP_Fare.xsd), _TripId_ (in _TripSummarStructure_), _FromTripIdRef_ and _ToTripIdRef_ (OJP_Fare.xsd). But I somehow don't understand the logic or intent of the _...IdRef..._ names.

Many ref point to a reference structure. But here, when it is just within the Trip I reference directly the ObjectIdType. I adapted it just as you suggested and fixed one that I missed before.

File: OJP/OJP_Fare.xsd [hunk 1]
```diff
@@ -208,22 +214,22 @@
 		</xs:annotation>
 		<xs:sequence>
 			<xs:element ref="FareProductRef"/>
-			<xs:element name="FromTripIdRef" type="ObjectIdType">
+			<xs:element name="FromTripIdRef" type="LegObjectIdType">
```
Comment (Aurige) [inline OJP/OJP_Fare.xsd:217]: Shouldn't it be TripObjectIdType ?

Comment (ue71603) [inline OJP/OJP_Fare.xsd:217]:
```suggestion
			<xs:element name="FromTripIdRef" type="TripObjectIdType">
```

Comment (ue71603) [reply OJP/OJP_Fare.xsd]: and now it is. tnx

**Review by Aurige — APPROVED** [review]

**Review by trurlurl — APPROVED** [review]

**Pull request merged**